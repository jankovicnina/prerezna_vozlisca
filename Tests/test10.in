43 210
9 13 0.11
3 5 2.87
2 40 -1.68
8 16 5.94
37 42 0.81
19 30 8.83
26 2 8.29
30 12 -1.8
38 31 -1.12
38 27 -0.67
5 14 2.36
4 31 3.67
17 25 7.7
12 2 7.76
37 31 8.46
13 18 4.38
10 16 6.9
38 14 2.45
8 21 4.99
15 0 1.89
27 35 2.98
33 1 9.51
10 19 1.73
40 34 -1.4
30 16 7.72
37 35 -1.66
34 8 2.15
42 4 5.1
33 36 2.03
4 42 2.77
31 4 2.79
32 21 -0.34
12 28 9.07
20 16 7.33
9 3 8.06
31 40 5.0
4 2 2.26
33 32 -1.4
22 21 1.88
1 10 5.14
34 28 5.55
0 39 3.51
21 9 3.34
30 1 8.41
26 5 8.34
36 32 -0.79
13 3 8.27
1 12 8.08
41 34 6.69
30 20 1.04
0 3 4.17
37 36 9.19
0 24 4.65
38 37 6.41
24 22 3.3
10 22 0.34
31 0 2.05
41 37 5.19
22 31 0.24
2 16 6.02
28 4 9.96
12 34 1.82
41 15 0.78
35 2 0.05
28 18 -1.7
2 5 5.8
13 23 -1.33
24 31 4.28
12 33 7.03
29 39 6.95
3 42 -1.96
15 5 5.2
16 23 4.57
31 2 1.82
30 21 2.85
7 35 3.6
14 1 -0.5
36 16 3.34
20 5 5.05
34 25 8.43
27 13 -1.69
24 13 0.95
8 37 9.03
20 12 2.86
14 28 4.2
3 33 0.82
5 18 7.63
41 11 7.65
27 23 4.59
12 5 8.09
8 32 9.61
33 21 -0.51
36 28 1.25
25 19 4.53
29 32 3.91
21 1 8.45
15 20 3.71
0 5 8.95
18 6 4.49
19 20 5.38
15 24 1.28
33 11 1.82
6 31 5.77
25 26 -1.81
4 28 7.1
42 0 4.5
37 5 5.19
20 0 7.24
40 13 0.73
13 6 8.28
25 1 4.33
11 35 0.04
10 38 1.28
24 10 -0.16
11 14 2.93
28 13 2.5
29 2 5.3
32 41 -0.32
39 8 8.14
39 41 2.69
33 6 0.26
13 20 9.3
18 39 -0.61
6 9 2.09
12 39 2.93
41 5 0.64
21 14 6.93
38 30 7.43
23 37 7.16
16 32 9.19
18 38 4.07
31 9 -0.59
7 9 -1.74
40 29 -1.39
2 33 8.99
12 15 2.49
42 31 7.04
0 40 0.49
31 27 6.73
26 6 5.5
19 2 8.89
26 27 6.69
37 7 1.72
23 1 4.07
33 28 0.72
24 3 1.62
6 10 7.38
22 8 3.08
10 4 5.91
10 36 9.0
27 22 4.79
12 27 8.0
38 11 0.32
29 4 8.7
10 25 0.4
30 39 5.87
35 22 6.01
5 32 2.54
13 14 3.75
7 36 9.11
17 26 8.82
22 4 4.14
24 40 8.53
37 13 3.48
34 9 4.28
42 13 2.72
26 38 -1.94
6 3 8.3
35 30 4.56
16 6 -1.86
38 13 -1.79
37 14 6.26
33 41 4.88
12 0 3.01
33 16 6.43
21 17 4.93
7 19 4.99
42 2 2.94
13 11 7.57
20 15 4.84
28 26 -1.55
6 0 -1.17
14 12 -0.44
19 21 6.42
17 10 -0.05
29 37 9.71
14 37 -2.0
25 32 5.83
23 21 -0.6
38 34 4.14
28 24 2.45
19 9 5.16
2 7 2.35
34 38 4.45
1 7 7.48
27 5 7.34
27 16 8.03
13 25 0.93
5 7 2.16
15 39 7.63
12 1 0.84
29 5 9.47
35 15 -0.64
31 16 -0.49
28 7 1.09
32 39 -1.69
16 27 0.18
25 16 0.63
23 19 7.17
30 25 -1.0
