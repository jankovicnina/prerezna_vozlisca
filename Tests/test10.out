1318 2104 3571 3895 4350 5828 8130 8468 8667 8987 9114 10856 10867 11953 12223 12361 13381 14143 14676 15167 15472 16910 17561 17613 18270 18337 19215 20716 21582
